test/emulation.spec.ts

```diff
@@ -408,4 +408,34 @@ describe('Emulation', () => {
       await page.emulateNetworkConditions(null);
     });
   });
+
+  describeFailsFirefox('Page.emulateCPUThrottling', function () {
```

Comment:
This is potentially very flaky so I don't think we should really test the effect. Let's just test that a command is received by the backend successfully.

Reply:
SGTM!